### ISSUE TITLE
service connection fails during api monitoring, and the monitoring list is displayed normally.

### ISSUE THREAD

**Issue by ShineDevelopment** [apache/hertzbeat#2054]
### Description

When I was monitoring the api, the display was normal after connecting the service, but it was normal after I stopped the service

**Comment by tomsun28** [comment]
Hi, is there has this api available alarm message in Alarm Center?

**Comment by ShineDevelopment** [comment]
> 你好，报警中心有这个api可以发送报警信息吗？

No, because the task status display is always normal, so the alarm center has no alarm data

**Comment by tomsun28** [comment]
maybe cause by https://github.com/apache/hertzbeat/issues/2062
